nmigen/build/generic_platform.py

```diff
@@ -0,0 +1,383 @@
+import edalize
```

Comment:
Can we do a lazy import so it's not a dependency for flows not using edalize?

Reply:
We've agreed (privately on IRC) to postpone edalize integration as edalize does not yet do what we want. I'm not sure why the PR isn't updated.